financialaid/models_test.py

```diff
@@ -22,3 +24,20 @@ def test_timestamped_model_update(self):  # pylint: disable=no-self-use
         Tier.objects.filter(id=tier.id).update(name="new_tier")
         third_timestamp = Tier.objects.get(id=tier.id)  # Since we need to re-fetch the object
         assert second_timestamp != third_timestamp
+
+    def test_financial_aid_model_unique(self):
```

Comment:
Can you also test that, if you update an existing `FinancialAid` you won't get the `ValidationError`?